src/main/java/newbank/server/commands/LogoutCommand.java

```diff
@@ -0,0 +1,24 @@
+package newbank.server.commands;
+
+import newbank.server.Customer;
+import newbank.server.CustomerID;
+import newbank.server.NewBank;
+
+public class LogoutCommand extends Command{
+    private final CustomerID customer;
+
+    public LogoutCommand(final NewBank bank, final String[] tokens, CustomerID customer){
+        this.customer = customer;
+    }
+
+    @Override
+    public String execute() {
+
+        if (!isLoggedIn(customer)) {
+            return "You are not logged in.";
```

Comment:
@claudist I would reuse the code from other Commands, the error messages should be the same.

Regards,
Jules